docs/branch-harvest/2026-05-03-release-v1.1.0-harvest-inventory.md

```
@@ -0,0 +1,127 @@
+---
+schema_version: 1
+title: Archscope release/v1.1.0 harvest inventory
+date: 2026-05-03
+source_branch: origin/release/v1.1.0
+base_branch: main
+status: complete
+---
+
+# Archscope release/v1.1.0 harvest inventory
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)